tests/pytests/test_themis_drivers.py

```diff
@@ -0,0 +1,397 @@
+                    ],
+                    [
+                        1.7514526413807416,
+                        1.7250030655476496,
+                        2.312291077819496
+                    ],
+                    [
+                        1.7535556099591456,
+                        1.7172463879119024,
+                        2.316431067882768
+                    ],
+                    [
+                        1.7556459015231667,
+                        1.7094774543366804,
+                        2.3205542750988553
+                    ]
+                ]
+            },
+            "SunPosition": {
+                "SpkTableStartTime": 589445681.51695,
+                "SpkTableEndTime": 589445681.51695,
+                "SpkTableOriginalSize": 1,
+                "EphemerisTimes": [
+                    589445681.51695
+                ],
+                "Positions": [
+                    [
+                        -177834612.75017235,
+                        94014077.38389017,
+                        47921572.85876171
+                    ]
+                ],
+                "Velocities": [
+                    [
+                        -13.267042406345718,
+                        -20.96669045246765,
+                        -9.258799613448325
+                    ]
+                ]
+            }
+          }
+        }
+      }
+
+
+@pytest.fixture()
+def test_themisir_kernels():
+    kernels = get_image_kernels('I74199019RDR')
+    updated_kernels, binary_kernels = convert_kernels(kernels)
+    yield updated_kernels
+    for kern in binary_kernels:
+        os.remove(kern)
+
+@pytest.fixture()
+def test_themisvis_kernels():
+    kernels = get_image_kernels('V46475015EDR')
+    updated_kernels, binary_kernels = convert_kernels(kernels)
+    yield updated_kernels
+    for kern in binary_kernels:
+        os.remove(kern)
+
+@pytest.mark.parametrize("label_type", ['isis3'])
+@pytest.mark.parametrize("formatter", ['isis'])
+def test_themisir_load(test_themisir_kernels, label_type, formatter):
+    label_file = get_image_label('I74199019RDR', label_type)
+    usgscsm_isd_str = ale.loads(label_file, props={'kernels': test_themisir_kernels}, formatter=formatter)
+    usgscsm_isd_obj = json.loads(usgscsm_isd_str)
+    print(json.dumps(usgscsm_isd_obj, indent=4))
+
+    assert compare_dicts(usgscsm_isd_obj, image_dict["I74199019RDR"][formatter]) == []
+
+@pytest.mark.parametrize("label_type", ['isis3'])
+@pytest.mark.parametrize("formatter", ['isis'])
+def test_themisvis_load(test_themisvis_kernels, label_type, formatter):
+    label_file = get_image_label('V46475015EDR', label_type)
+    usgscsm_isd_str = ale.loads(label_file, props={'kernels': test_themisvis_kernels}, formatter=formatter)
+    usgscsm_isd_obj = json.loads(usgscsm_isd_str)
+    print(json.dumps(usgscsm_isd_obj, indent=4))
+
+    assert compare_dicts(usgscsm_isd_obj, image_dict["V46475015EDR"][formatter]) == []
```

Comment:
These two load tests can be parameterized into one, see voyager:

https://github.com/USGS-Astrogeology/ale/blob/master/tests/pytests/test_voyager_drivers.py#L302